i18n/en.json

```diff
@@ -678,6 +678,8 @@
   "admin.guest_access.enableDescription": "When true, external guest can be invited to channels within teams. Please see [Permissions Schemes](../user_management/permissions/system_scheme) for which roles can invite guests.",
   "admin.guest_access.enableTitle": "Enable Guest Access: ",
   "admin.guest_access.mfaDescription": "When true, [multi-factor authentication](!https://docs.mattermost.com/deployment/auth.html) for guests is required for login. New guest users will be required to configure MFA on signup. Logged in guest users without MFA configured are redirected to the MFA setup page until configuration is complete.\n \nIf your system has guest users with login methods other than AD/LDAP and email, MFA must be enforced with the authentication provider outside of Mattermost.",
+  "admin.guest_access.mfaDescriptionMFANotEnabled": "[multi-factor authentication](./mfa) is currently not enabled.",
+  "admin.guest_access.mfaDescriptionMFANotEnforced": "[multi-factor authentication](./mfa) is currently not enforced.",
```

Comment:
Please capitalize Multi-factor on both of these as well.

Reply:
Done